rhobs/alerting/data_plane/prometheus.availability_metric_konflux_alerts.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-konflux-up-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: konflux_up_alerts
+    interval: 1m
+    rules:
+    - alert: KonfluxAlert
+      expr: (konflux_up offset 1h) unless on(service, check, source_cluster) konflux_up
+      for: 1m
```

```suggestion
      for: 5m
```
I suspect `1m` might be too sensitive and therefore result in a lot of alert flapping.